Public/New-HTMLTab.ps1

```diff
@@ -68,7 +68,8 @@ function New-HTMLTab {
         [parameter(ParameterSetName = "FontAwesomeSolid")][int] $IconSize,
         [parameter(ParameterSetName = "FontAwesomeBrands")]
         [parameter(ParameterSetName = "FontAwesomeRegular")]
-        [parameter(ParameterSetName = "FontAwesomeSolid")][string] $IconColor
+        [parameter(ParameterSetName = "FontAwesomeSolid")][string] $IconColor,
+        [ValidateSet('uppercase', 'lowercase', 'capitalize')][string] $TextTransform  # New-HTMLTab - Add text-transform
```

Comment:
Why not define = 'uppercase' as default here so you can skip if ($TextTransform) ? Also the default now is "none" right? With your change you want uppercase if nothing is selected?

Reply:
Good catch! Yes, I wanted to keep the 'uppercase' as the default and set to other values if selected.